### PR TITLE
Amir yunas

### PR DESCRIPTION
**Pull request by ayunas**
@wcolts2000 

I built out this project from scratch, wanting to get those reps in.  I also added styling and imagery to make it more fun. (mostly for my son, who was excited I was coding a star wars project).  

Comments are always appreciated.  Thanks!

### CODE REVIEW

**Comment by ayunas** [comment]
We got sound effects!
@wcolts2000 
live project:  starwars.lambdaschool.me

File: starwars/package.json [hunk 1]
```diff
@@ -3,9 +3,19 @@
...
+    "axios": "^0.18.0",
+    "node-sass": "^4.11.0",
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.0"
+    "react-loader-spinner": "^2.3.0",
+    "react-redux": "^7.0.2",
+    "react-scripts": "3.0.0",
+    "react-sound": "^1.2.0",
```
Comment (wcolts2000) [inline starwars/package.json]: I haven't played with this one yet, I'll give it a look this weekend! Nice job

File: starwars/public/index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>SWAPI</title>
```
Comment (wcolts2000) [inline starwars/public/index.html]: Yes!!! You'd be surprised how many people don't change this

File: starwars/src/App.js [hunk 3]
```diff
@@ -1,26 +1,56 @@
...
+        {c3po : 'http://bit.ly/2DARP81'},
+        {r2d2 : 'http://bit.ly/2PwSk7H'},
+        {vader: 'http://bit.ly/2ZCX68C'},
+        {leia : 'http://bit.ly/2DzE21g'},
+        {owen : 'http://bit.ly/2GFq8LN'},
+        {beru : 'http://bit.ly/2PwSV9r'},
+        {r5d4 : 'http://bit.ly/2XL3Yia'},
+        {biggs: 'http://bit.ly/2PysXCK'},
+        { obi : 'http://bit.ly/2vlO1Tw'}
+    ]
```
Comment (wcolts2000) [inline starwars/src/App.js]: very cool

File: starwars/src/App.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (wcolts2000) [inline starwars/src/App.scss]: you can go through and delete the boilerplate styles in here

File: starwars/src/Footer.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    <footer>
+       <figure>
+            <div>
+                <p>Built with: </p>
+                <img src={redux} />
+            </div>
+
+            <div>
+                <p>Project of: </p>
+                <a href='http://www.lambdaschool.com' target="_blank"> <img className='lambda' src={lambdaLogo} /></a>
```
Comment (wcolts2000) [inline starwars/src/Footer.js]: you remove the console.warning when you have the target _blank attribute, you will want to add a rel="noopener noreferrer" attribute the the a tag also. this is a security thing regarding the opened window still linking back to your site and the possibility of XSS

File: starwars/src/Header.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+// import Loader from 'react-loader-spinner';
+import './App.scss';
+
+
+function Header() {
+  return (
+    <header className="App-header">
+        {/* <Loader type="Grid" color="white" />   */}
+        {/* <h2> SWAPI </h2> */}
+        <a href='https://swapi.co' target='_blank'><img className='starwars' src='http://bit.ly/2IIuqWk' /></a>
```
Comment (wcolts2000) [inline starwars/src/Header.js]: same note as above on the anchor tags attributes regarding the target _blank here

File: starwars/src/StarWars.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+function beam() {
+    console.log('you have beamed');
+    const beam = new Audio(lightsaber);
+    beam.play();
+}
+
+function StarWars(props) {
+    console.log('avatars props: ' , props.avatars);
+    const urls = props.avatars.map( jedi => Object.values(jedi) );
```
Comment (wcolts2000) [inline starwars/src/StarWars.js]: interesting way to set the avatars

File: starwars/src/StarWars.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+                <div>
+                    <span>Loading... Please Wait...</span> 
+                    <Loader type="Circles" color="navy"/>
+                </div> 
+                }
+                
+                { props.starwars.map( (jedi,i) => {
+                        return (
+                        <div className='jedi'>
+                            <img onClick={beam} src={urls[i][0]} />
```
Comment (wcolts2000) [inline starwars/src/StarWars.js]: i like the url variables you are using here

**Pull request merged**